### PR TITLE
Documentation update

### PR DESCRIPTION
**Pull request by tomaszrybicki**
Signed-off-by: Tomasz Rybicki <tomasz.rybicki@intel.com>

### CODE REVIEW

File: doc/IOTRACER.md [hunk 1]
```diff
@@ -31,28 +31,24 @@ this instruction. Let's get a trace. Any time you can call the tool's help:
...
-     -S    --start-trace                         Start IO tracing
-     -V    --version                             Print version
+     -H    --help                                Prints help
+     -S    --start-tracing                       Starts IO tracing
+     -V    --version                             Prints version
+
 ~~~
 
-We have to invoke _--start-trace_ command. Its help is: 
+We have to invoke _--start-tracing_ command. Its help is:
```
Comment (arutk) [inline doc/IOTRACER.md]: This line sounds awkward. I would rephrase:

To start trace collection use --start-tracing command. Command usage:

File: doc/man/MAN.md [hunk 2]
```diff
@@ -1,7 +1,7 @@
 # iotrace - collect and manage block I/O traces
 
 ```
-iotrace --start-trace -d device[,device,...] [-b buffer_size] [-s max_file_size] [-t seconds]
+iotrace --start-tracing -d device[,device,...] [-b buffer_size] [-s max_file_size] [-t seconds]
 iotrace --parse-trace [--format json] --path path
 iotrace --get-trace-summary --path path
 iotrace --list-traces [--prefix prefix]
```
Comment (Ostrokrzew) [inline doc/man/MAN.md]: You could update this commands too.

File: doc/IOTRACER.md [hunk 3]
```diff
@@ -120,7 +117,7 @@ Options that are valid with {-P | --parse-trace}
 To get CSV, call:
 
 ~~~{.sh}
-iotrace --parse-trace --path kernel/2019-08-13_12:35:22 --format csv
+iotrace --trace-parsing --parse-trace --path kernel/2019-08-13_12:35:22 --format csv
```
Comment (mariuszbarczak) [inline doc/IOTRACER.md]: Please update it according to new CLI interface

File: doc/IOTRACER.md [hunk 4]
```diff
@@ -103,11 +99,12 @@ processing.
 ## Parsing traces
 
 The iotrace parser converts IO traces from binary format to CSV or JSON format.
-To parse you trace we are going to invoke _--parse-trace_ command.
+To parse you trace we are going to invoke _--parse-trace_ command from
```
Comment (mmkayPL) [inline doc/IOTRACER.md]: **your** trace
